desktop/libs/notebook/src/notebook/templates/editor_components.mako

```diff
@@ -221,7 +221,7 @@ else:
   % endif
 
     <div class="btn-group">
-      <a class="btn" rel="tooltip" data-placement="bottom" data-loading-text="${ _("Saving...") }" data-bind="click: function() { if ($root.canSave() ) { saveNotebook() } else { $('#saveAsModal${ suffix }').modal('show');} }, attr: { title: $root.canSave() ? '${ _ko('Save') }' : '${ _ko('Save As') }' }">
+      <a class="btn" data-testid="editor--save-as--button" rel="tooltip" data-placement="bottom" data-loading-text="${ _("Saving...") }" data-bind="click: function() { if ($root.canSave() ) { saveNotebook() } else { $('#saveAsModal${ suffix }').modal('show');} }, attr: { title: $root.canSave() ? '${ _ko('Save') }' : '${ _ko('Save As') }' }">
```

Comment:
editor--save--button (the same is used for both "Save As" and "Save")

Reply:
Vishnu mentioned it as"title="Save As" (Save Button)" so I added it. Should I remove it from here?

Reply:
Just rename to editor--save--button and it's fine.